homeassistant/components/vesync/sensor.py

```diff
@@ -186,6 +190,15 @@ def ha_dev_type(device):
         update_fn=update_energy,
         exists_fn=lambda device: ha_dev_type(device) == "outlet",
     ),
+    VeSyncSensorEntityDescription(
+        key="humidity",
+        translation_key="current_humidity",
```

Comment:
translation key can be omitted as it has a device class so it will use the device class name

Reply:
Hello @joostlek, is it a practice to expose humidity level as a separate sensor when humidifier exposes the same as an attribute? 

I looked at some other integrations like Xiaomi and they don't expose additional sensor. And one can always create a templated entity if needed.